forem/forem
# Move conditional to render dots only on published articles

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents


- Closes #19305
## QA Instructions, Screenshots, Recordings

-Go to http://localhost:3000/new
-Add a title
-Press on save draft
-Mod icon and dots should not be visible
-Click on edit
-Click on publish
-Mod  icon, dots should be visible

Before changes published articles have the dots and ui icons on the left:
![before-published](https://user-images.githubusercontent.com/54085409/235361169-cc35b5d2-fc12-4634-8968-f4bf2b303168.png)

After changes, published articles still have them:
![after-published](https://user-images.githubusercontent.com/54085409/235361355-8a8c756e-ec55-41d9-8979-64486fc4dfc8.png)

Before changes, not published articles/draft articles also have these ui icons on the let:
![before-not-published](https://user-images.githubusercontent.com/54085409/235361327-7713b144-1efa-439b-baea-df10a702f8f4.png)

After changes, not published articles/draft articles do not display these icons anymore:
![after-not-published](https://user-images.githubusercontent.com/54085409/235361688-951096c3-bf20-462d-a165-3e87d4149182.png)



## Added/updated tests?

      have not been included_
- [x ] I need help with writing tests: Running on the project on docker, when I run yarn e2e I get the following message.
-  [1/6] Validating package.json...
error dev.to@1.0.0: The engine "yarn" is incompatible with this module. Expected version "1.22.18". Got "1.22.17"
error Found incompatible module.
